heft-plugins/heft-webpack5-plugin/README.md

```diff
@@ -0,0 +1,13 @@
+# @rushstack/heft-webpack5-plugin
+
+> 🚨 _This is an early preview release. Please report issues!_ 🚨
```

Comment:
Heft itself dropped this notice a while ago.  It seems unnecessary.

Reply:
@iclanton Also see [this recent Zulip discussion](https://rushstack.zulipchat.com/#narrow/stream/262513-general/topic/rush-roadmap/near/232495556):

**Hollis Mendel:**
> Thanks a lot for answering that @**Pete Gonzalez** . I work in a largish team and agree that we've faced a lot of what rush tries to solve. Appreciate you and the team taking time out to contribute.
> 
> To me externally, however, there seems to be a lot of churn happening, and rush appears to support a lot of things in an experimental state.
> For eg, support for npm / yarn, remote caching, watch mode, heft etc — is rush working on too many things all at once vs. solidifying its existing feature set?
> 

**Pete Gonzalez:**
> Hmm...  that's interesting feedback.  Maybe the docs/roadmap should address that somehow.
> 
> Probably different people would have different opinions. Here's my thoughts:
> 
> The Rush maintainers serve two worlds:  (1) your own company's monorepo and (2) the OSS community.  Major features generally get funded by (1) before we document and support them for (2).  This means that many of Rush's "experimental" features are already getting heavy usage by serious production monorepos.  So "experimental" often does not imply that it's buggy (or even incomplete), but rather that we reserve the right to make breaking changes to the design based on what we learn from firsthand usage.
> 
> But it depends on the feature.  For the examples you gave:
> - npm/yarn -- nobody is using that any more, so it's not very stable
> - cloud build cache -- Microsoft has been using this at scale since January, so it's by now fairly mature, although we didn't publish the docs for it yet (that's my fault; I write a lot of the docs and have been super busy lately)
> - watch mode -- the features implemented so far work reasonably well, but there are [4 important work items](https://github.com/microsoft/rushstack/issues/1202#issuecomment-740420600) that must be finished before it can be called a fully complete solution.
> - heft -- it's pretty mature by now and is used in very large repos. It's only "experimental" in the sense that we will probably make some breaking design changes to the CLI and config files before releasing a "1.0" version.
> 
> Maybe it would be useful for the roadmap to call out these subprojects and track their status more visibly. We also sometimes forget to graduate them (for example `rush deploy` was designated as `experimental` long after its design had stabilized).
>

Reply:
Good call. Removed.